pages/common/dolt-merge.md

```diff
@@ -0,0 +1,24 @@
+# dolt merge
+
+> Join two or more development histories together.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Incorporate changes from the named commits (since the time their histories diverged from the current branch) into the current branch:
```

Comment:
The branch could be up to date and the command doesn't fail. Also I think we should explain a merge in this page
```suggestion
- Incorporate changes from the named commits into the current branch:
```

Reply:
solved in 55a842f39a01070f08987fc752e5ab667f8e96ef